src/OpenTelemetry.Instrumentation.Owin/MetricProviderBuilderExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="MetricProviderBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Instrumentation.Owin.Implementation;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Metrics;
+
+/// <summary>
+/// Extension methods to simplify registering of OWIN request instrumentation.
+/// </summary>
+public static class MetricProviderBuilderExtensions
```

Comment:
nit: Would you mind renaming this something like `OwinInstrumentationMeterProviderBuilderExtensions`? We have a lot of stuff generically named like `MetricProviderBuilderExtensions` in shared namespaces like `OpenTelemetry.Metrics` it isn't the end of the world but it creates problems for some tools like documentation generators.